src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/validation/FormAValidator.java

```diff
@@ -200,4 +222,72 @@
                 ? asList("HWFNumber is usually 6 digits")
                 : emptyList();
     }
+
+    public OcrValidationResult validateFormAScannedDocuments(ExceptionRecord exceptionRecord) throws UnsupportedFormTypeException {
+
+        List<InputScannedDoc> scannedDocuments = exceptionRecord.getScannedDocuments();
+        List<InputScannedDoc> inputScannedDocs = Optional.ofNullable(scannedDocuments).orElse(emptyList());
+        OcrValidationResult.Builder documentValidationResultBuilder = OcrValidationResult.builder();
+
+        List<String> validationMessagesForScannedDocuments = produceErrorsForIncorrectNumberOfAttachedDocuments(inputScannedDocs);
+        validationMessagesForScannedDocuments.forEach(documentValidationResultBuilder::addWarning);
+
+        List<String> validationMessagesForDocumentSubTypeNotAccepted = produceErrorsForDocumentSubTypeNotAccepted(inputScannedDocs);
+        validationMessagesForDocumentSubTypeNotAccepted.forEach(documentValidationResultBuilder::addWarning);
+
+        return documentValidationResultBuilder.build();
+    }
+
+    private List<String> produceErrorsForIncorrectNumberOfAttachedDocuments(List<InputScannedDoc> inputScannedDocs) {
+
+        /*
+        Validates that only one 'Form A' and one 'Draft Consent Order' are attached to the Exception Record
+        Validates that there is at least one 'D81' attached to the Exception Record
+         */
```

Comment:
can make this as well JavaDoc entry - start with `/**` above method declaration and IntelliJ will autocomplete.